Mage.Sets/src/mage/cards/c/ChandrasRegulator.java

```diff
@@ -68,56 +65,11 @@ public ChandrasRegulator copy() {
     }
 }
 
-class ChandrasRegulatorTriggeredAbility extends TriggeredAbilityImpl {
```

Comment:
Refactored this into `ActivatePlaneswalkerLoyaltyAbilityTriggeredAbility` which can handle different subtypes of PWs